### DISCUSSION TITLE
Frictionless

### DISCUSSION THREAD

**Discussion by dennisangemi** [ondata/guidaPraticaPubblicazioneCSV#83] [Idee/Proposte]
Buongiorno e grazie per lo splendido lavoro!

Frictionless può essere utilizzato per realizzare un data dictionary, giusto? Perchè non inseriamo allora un piccolo paragrafetto nella sezione "[Dizionari di dati](https://ondata.github.io/guidaPraticaPubblicazioneCSV/guida/dizionario_dati/)" che faccia riferimento al datapackage di frictionless e a quanto sia intuitivo e facile da implementare?

Aggiungo una domanda banale che nasce da una curiosità personale: è "meglio" avere un data dictionary come quello descritto [qui](https://ondata.github.io/guidaPraticaPubblicazioneCSV/guida/dizionario_dati/#esempio-di-dizionario-dati-in-formato-json-ld) o un datapackage di frictionless?

**Comment by aborruso** [comment]
Caro @dennisangemi,

> Buongiorno e grazie per lo splendido lavoro!

grazie mille e grazie per il tuo contributo.

> Frictionless può essere utilizzato per realizzare un data dictionary, giusto? Perchè non inseriamo allora un piccolo paragrafetto nella sezione "[Dizionari di dati](https://ondata.github.io/guidaPraticaPubblicazioneCSV/guida/dizionario_dati/)" che faccia riferimento al datapackage di frictionless e a quanto sia intuitivo e facile da implementare?

Come sai amo molto questo standard, ma il primo obiettivo di questo lavoro, che dobbiamo ancora lanciare, è tradurre la guida originale. La guida ha qualche anno di età e sarà da aggiornare, rispetto a standard, strumenti, ecc. e quello del frictionless sarà certamente un'integrazione.
Se vuoi iniziare a scrivere una sezione dedicata, non esitare.


> Aggiungo una domanda banale che nasce da una curiosità personale: è "meglio" avere un data dictionary come quello descritto [qui](https://ondata.github.io/guidaPraticaPubblicazioneCSV/guida/dizionario_dati/#esempio-di-dizionario-dati-in-formato-json-ld) o un datapackage di frictionless?

Rispetto a quello, oggi lo standard frictionless è più maturo, quindi sceglierei quest'ultimo. Quello che conta di più è descrivere i propri dataset.

Creo una issue di enhancement  da questo tuo messaggio.

Grazie ancora

**Reply by dennisangemi** [reply]
Grazie Andrea, troverò del tempo per scrivere qualcosa ;)
